apps/playground/src/app/modules/components/tile-module/tile-module.component.html

```diff
@@ -65,168 +64,220 @@ <h2>DsfrTileComponent</h2>
       <mat-icon>open_in_new</mat-icon>
     </a>
     </play-input-property-definition>
+    <play-input-property-definition
```

Comment:
```html
<play-input-property-definition
      name="detail"
      type="It takes a string">
      An optional text to help end-users understand the purpose of a tile.
      <br />
      ⚠️Note that it will NOT be displayed when the
      <code>download</code> property has a value. Instead, we will show the
      format, the weight and optionally, a description, for the file to be
      downloaded.
    </play-input-property-definition>
    <play-input-property-definition
      name="download"
      type="DownloadOptions">
      It is used to enable the end-users to download a file. The value must
      satisfy the <code>DownloadOptions</code> interface :
      <pre>
<span class='highlight-code'>export interface DownloadOptions</span> &#123;
    open?: boolean; <span class='highlight-comment'>// Whether or not you want the file to be downloaded and opened right away</span>
    openOnAnotherPage?: boolean; <span class='highlight-comment'>// Whether or not you want the file to be opened on another page</span>
    fileName?: string; <span class='highlight-comment'>// The name of the file when downloaded. Leave empty to keep the name from the server</span>
    fileLang?: string; <span class='highlight-comment'>// The language of the downloaded file</span>
    fileDesc?: string; <span class='highlight-comment'>// An optional description of the file</span>
    fileType: string; <span class='highlight-comment'>// The extension of the file</span>
    fileWeight: string; <span class='highlight-comment'>// The expected weight for the file, ex: '42 Ko'</span>
&#125;</pre>
      ⚠️ Please be aware that <code>fileType</code> and
      <code>fileWeight</code> are mandatory.<br />
      By default, the other properties of the interface, if not provided, will
      be set to the values :
      <pre>
&#123;
    open: false,
    openOnAnotherPage: true,
    fileName: '',
    fileLang: 'fr'
&#125;</pre
      >
      The property
      <code>link</code> must hold the path of the corresponding file. The
      property <code>title</code> will be prefixed with the mention
      "Télécharger"
    </play-input-property-definition>
```